changelogs/65752-fix-azure_rm_storageblob-typo

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
Sorry, but could you please add file extension (.yml or .yaml) to this file?

Reply:
Done. Sorry that I had overlooked that!